app/helpers/application_helper.rb

```diff
@@ -45,4 +45,12 @@ def error_messages_for(model, field)
       end
     end
   end
+
+  def product_name
+    @product_name ||= Rails.root.join("PRODUCT").read
+  end
+
+  def product_version
+    @product_version ||= Rails.root.join("VERSION").read
+  end
```

Comment:
Instead of reading this on a helper every time I think we should save this using an initializer (http://guides.rubyonrails.org/configuring.html#custom-configuration), so we only read this once when we start the process.

Reply:
good point, updated